DigitalLearningSolutions.Data/Models/DiagnosticAssessment/OldTutorial.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace DigitalLearningSolutions.Data.Models.DiagnosticAssessment
+{
+    public class OldTutorial
```

Comment:
Don't we already have a class like this? Or am I thinking of one @SChandarana has added in an open PR?

Reply:
Yep I have "OldSectionTutorial.cs" in PR #110 which is the same as this

Reply:
In that case whichever one of you merges last should rebase and use the existing class. I'll leave it up to you to coordinate that 🙂 @SChandarana I think OldTutorial might be a better name 🙂

Reply:
We've sorted out the order for this, are there any other comments for this PR?